### ISSUE TITLE
Question about Test Results

### ISSUE THREAD

**Issue by hubzyx** [tegusi/EAGRNet#9]
Thanks for your great work and kind sharing.

I try to reproduce your work on CelebAMask dataset, 24,183 images were selected as the training set and 2834 as the test set. Then I generated label images, edge images and txt files.

After that, I used the code you provided for training. I did not modify the training parameters, but after the training, I found that my training results were nowhere near as good as yours:

Pixel accuracy: 88.680001 
Mean accuracy: 62.602909 
Mean IU: 52.106180 
f1_bg=0.9206227882646921
f1_skin=0.9063355322233573
f1_nose=0.8564767842156585
f1_eye_g=0.7503944737085695
f1_le=0.5846867698606126
f1_re=0.6236390541383516
f1_lb=0.5334376035132539
f1_rb=0.5940682964151555
f1_l_ear=0.5788707588593692
f1_r_ear=0.6270729442117574
f1_imouth=0.6891172758840716
f1_ulip=0.6324672234653816
f1_llip=0.7040783923686705
f1_hair=0.9064766548039499
f1_hat=0.7394414853009562
f1_ear_r=0.23874973724403548
f1_neck_l=0.0
f1_neck=0.812055395815203
f1_cloth=0.7392161008030024
f1_eyes=0.6047181516247372
f1_brows=0.5652373988652979
f1_mouth=0.8338039315469467
f1_overall=0.7834263360796998

The loss in my training have fluctuated between 1.5 and 2.1. And I checked the data set to make sure it had no errors.  During the training process, I directly copied your open source code and trained it on train.py. I would like to consult you whether I neglected some key steps? And how can I achieve your results as best as possible? 

I would be very grateful if you could reply to me.

**Comment by tegusi** [comment]
It is mostly because we use different face alignment methods. The face alignment in our framework is not public due to copyright issues. You could preprocess the alignment matrices by OpenCV to see if the results improved.

**Comment by Wu-ZJ** [comment]
你好，请问name list是怎么样的呢？
